Makefile

```
@@ -42,6 +42,7 @@ package-chart-%: lint-chart-%
 
 .PHONY: helm-package
 helm-package: $(CHARTS_PACKAGE_DIR) $(EXTENSION_CHARTS_PACKAGE_DIR)
+	rm -rf $(LOCALBIN)/charts
```

There is `CHARTS_PACKAGE_DIR ?= $(LOCALBIN)/charts` above, so I guess we should use it here:
```suggestion
	rm -rf $(CHARTS_PACKAGE_DIR)
```